lib/OpenQA/Setup.pm

```diff
@@ -396,7 +396,7 @@ sub load_plugins ($server, $monitoring_root_route = undef, %options) {
 }
 
 sub set_secure_flag_on_cookies ($c) {
-    $c->app->sessions->secure(1) if $c->req->is_secure;
+    $c->app->sessions->encrypted(1) if $c->req->is_secure;
```

Comment:
These two are not related, `secure` makes the sessions cookie require HTTPS and the `is_secure` check ensures HTTPS was used for the request.